debian/rules

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@
+
+override_dh_auto_test:
+	echo "FIXME: We need to fix gtest build for the RPi to run dh_auto_test"
```

Comment:
We might end up not even running the tests here as they will be tested elsewhere and up to now we haven't done this in this step but given how we want this to be entirely cross-platform, we might actually want to do this here.

Reply:
I will leave the `echo` as it is then, meanwhile we clarify how to best progress